### DISCUSSION TITLE
getting too many contigs and as many scaffolds

### DISCUSSION THREAD

**Discussion by aayushirambia** [ablab/spades#1041] [Q&A]
Hi,
I am trying to assemble a genome and getting 1234 contigs and scaffolds. I tried using different k-mer values, but as I kept on increasing the k-mer values to uptil, the number of contigs kept on increasing.
How do I resolve this issue to get less no of contigs?
The reference genome has 2 chromosomes and a few plasmids. Is there some other method that I should use for assembly then?

**Comment by asl** [comment]
So, why do you think it's "too many"? What is the genome size? Read fragment length / insert size? Repeat contents?
